params/Lukasz's_bird.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2.1245
+    
+# e.g. "anas_krystallinus"    
+species.name <- " Lukasz's penguin "
+    
+# e.g. "red"    
+color <- " purple "
```

Comment:
Ditto for this line...

```suggestion
color <- "purple"
```